Aliases/utica@0.40

```diff
@@ -0,0 +1 @@
+../Formula/utica.rb
```

Comment:
Could you please add some context on why do we need this?

Reply:
The test-bot Action was giving the following error:

```
Formula has other versions so create a versioned alias:
  cd /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/interfere/homebrew-outdated/Aliases
  ln -s ../Formula/utica.rb utica@0.40
  ```